test/widgets/content_test.dart

```diff
@@ -469,7 +526,84 @@ void main() {
     check((ratioInHeader - ratioInParagraph).abs()).isLessThan(0.001);
   }
 
-  testContentSmoke(ContentExample.strong);
+  group('strong (bold)', () {
+    testContentSmoke(ContentExample.strong);
+
+    TextStyle findWordBold (tester) {
+      final root = tester.renderObject<RenderParagraph>(find.textContaining('bold')).text;
+      return mergedStyleOfSubstring(root, 'bold')!;
+    }
+
+    testFontWeight('in plain paragraph',
+      expectedWght: 600,
+      // **bold**
+      content: plainContent('<p><strong>bold</strong></p>'),
+      styleFinder: findWordBold,
+    );
+
+    for (final level in HeadingLevel.values) {
+      final name = level.name;
+      assert(RegExp(r'^h[1-6]$').hasMatch(name));
+      testFontWeight('in $name',
+        expectedWght: 800,
+        // **bold**
+        content: plainContent('<$name><strong>bold</strong></$name>'),
```

Comment:
This comment is inaccurate because there's the heading element.

Fine to have this automatic generation of six test cases, just need to adjust the comment. E.g.:
```suggestion
        // # **bold**, ## **bold**, ### **bold**, etc.
        content: plainContent('<$name><strong>bold</strong></$name>'),
```